### PR TITLE
Add translation for satpam

### PR DESCRIPTION
**Pull request by albertstevelino**

### CODE REVIEW

File: src/locale/id.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  'alpha': 'Input harus berupa huruf.',
```
Comment (febrianyuwono) [inline src/locale/id.js:2]: hi @albertstevelino , these validations is missing 
```fileType:$1
not-equal:$1
not-memberOf:$1
date
fqdn
imei
string

**Review by febrianyuwono — CHANGES_REQUESTED** [review]

File: src/locale/id.js (hunk 1 above)
Comment (albertstevelino) [reply src/locale/id.js:2]: thanks for the notes

File: src/locale/id.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  'alphanumeric': 'Input harus berupa huruf atau angka.',
+  'beginWith:$1': 'Input tidak valid.',
+  'between:$1:$2:': 'Input antara <%= ruleParams[0] %> dan <% ruleParams[1] %>.',
+  'boolean': 'Input harus boolean.',
+  'creditCard': 'Nomor kartu kredit tidak valid.',
+  'dateAfter:$1:$2:$3:$4': 'Tanggal tidak valid.',
+  'dateBefore:$1:$2:$3:$4': 'Tanggal tidak valid.',
+  'dateFormat:$1': 'Format tanggal tidak valid.',
+  'email': 'Format email tidak valid.',
+  'emptyString': 'Input harus kosong',
```
Comment (osumampouw) [inline src/locale/id.js:13]: ini bukannya input harus diisi?

Comment (albertstevelino) [reply src/locale/id.js:13]: kaga ko, emang harus kosong
`const validate = R.equals('');`

**Review by febrianyuwono — APPROVED** [review]

**Pull request merged**